### ISSUE TITLE
Send the comments

### ISSUE THREAD

**Issue by mshafiei** [Fillll/reddit2telegram#63]
Hi :)

Is there an easy way to send every comment in the subreddit?

**Comment by wiz64** [comment]
No, the developers haven't added such features to this yet, its just for the posts currently.

**Comment by Fillll** [comment]
> Hi :)
> 
> Is there an easy way to send every comment in the subreddit?

Could you explain how you see sending comments in the target state?
